Tests/Remove-JiraIssue.Tests.ps1

```diff
@@ -0,0 +1,272 @@
+Describe "Remove-JiraIssue" {
+    Import-Module "$PSScriptRoot/../JiraPS" -Force -ErrorAction Stop
+
+    InModuleScope JiraPS {
+
+        . "$PSScriptRoot/Shared.ps1"
+
+        $script:ShowMockData = $true
```

Comment:
Please remove this for the PR once you are done with debugging

Reply:
Whoops! I completely forgot I'd left that in there. Commit 157dcd4b09b449b79639e6c1a0b3738a4995cbf9 removed that line.